### PR TITLE
Implement ProcessProposerSlashing

### PR DESCRIPTION
**Pull request by michaelneuder**
Spec: https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#proposer-slashings

Part of https://github.com/ledgerwatch/erigon/issues/5965

### CODE REVIEW

File: cmd/erigon-cl/core/transition/beacon_state_utils.go [hunk 1]
```diff
@@ -142,6 +143,15 @@ func ComputeSigningRootEpoch(epoch uint64, domain []byte) ([32]byte, error) {
 	return utils.Keccak256(hash[:], domain), nil
 }
 
+func ComputeSigningRoot(obj ssz_utils.ObjectSSZ, domain []byte) ([32]byte, error) {
```
Comment (Giulio2002) [inline cmd/erigon-cl/core/transition/beacon_state_utils.go:146]: I think we already have this in fork package

Comment (Giulio2002) [reply cmd/erigon-cl/core/transition/beacon_state_utils.go:146]: fork.ComputeSigningRoot

**Comment by Giulio2002** [comment]
lint is broken btw, otherwise LGTM

File: cmd/erigon-cl/core/transition/beacon_state_utils.go (hunk 1 above)
Comment (michaelneuder) [reply cmd/erigon-cl/core/transition/beacon_state_utils.go:146]: thanks, done!

**Review by Giulio2002 — APPROVED** [review]

**Comment by Giulio2002** [comment]
will rerun ubuntu CI first :)

**Pull request merged**